cryoet_data_portal_neuroglancer/models/json_generator.py

```diff
@@ -197,25 +191,20 @@ def generate_json(self) -> dict:
 class ImageVolumeJSONGenerator(RenderingJSONGenerator):
     """Generates JSON Neuroglancer config for volume rendering."""
 
-    color: str
-    rendering_depth: int
+    rendering_depth: int  # Ideally, this should be a power of 2
+    contrast_limits: tuple[float, float] = (-64, 64)
+    threedee_contrast_limits: tuple[float, float] = (-64, 64)
     is_visible: bool = True
 
     def __post_init__(self):
         self._type = RenderingTypes.IMAGE
 
     def _get_shader(self) -> dict[str, Any]:
-        shader = (
-            f'#uicontrol vec3 color color(default="{self.color}")\n'
-            f"#uicontrol invlerp toRaw(range=[0, 1], window=[-1, 2])\n"
-            f"void main() {{\n"
-            f"  emitRGBA(vec4(color * toRaw(getDataValue()), toRaw(getDataValue())));\n"
-            f"}}"
+        shader_builder = ImageShaderBuilder(
```

Comment:
This class was created for rendering the segmentation as 3d volume, I think, it would be more impactful to apply this shader to the `ImageJSONGenerator` class.

Reply:
Ah I see, thanks Manasa! I'll remove the changes from this class